### PR TITLE
Basic UI of schedule payment form

### PR DESCRIPTION
**Pull request by lukemelia**
This PR contains basic UI for the schedule payment form. I think we can review and merge it now and continue work on individual aspects and wiring it to services/data.

image

### CODE REVIEW

File: packages/boxel/addon/components/boxel/action-chin/usage.gts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default class ActionChinUsage extends Component<Signature> {
   @tracked stepNumber = 0;
   @tracked state: ActionChinState = 'default';
   @tracked disabled = false;
+  @tracked hideLock = false;
```
Comment (backspace) [inline packages/boxel/addon/components/boxel/action-chin/usage.gts:29]: Is this used anywhere?

File: packages/boxel/addon/components/boxel/action-container/usage.gts [hunk 2]
```diff
@@ -57,6 +58,13 @@ export default class extends Component {
           @optional={{true}}
           @description="Unstyled area for custom card content and fields; yields a Section component that takes an optional title and applies an appropriate inset"
         />
+        <Args.Bool
+          @name="hideLock"
+          @description="When true, suppress the 'Actions only visible to you' message and lock icon in the chin."
+          @defaultValue={{false}}
+          @value={{this.hideLock}}
+          @onInput={{fn (mut this.hideLock)}}
+        />
```
Comment (backspace) [inline packages/boxel/addon/components/boxel/action-container/usage.gts:67]: We discussed removing this entirely, right?

File: packages/safe-tools-client/app/components/safe-tools/schedule-payment-form-action-card/index.gts [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+            </ac.ActionButton>
+          </:default>
+        </ActionChin>
+    </BoxelActionContainer>
+  </template>
+}
+
+declare module '@glint/environment-ember-loose/registry' {
+  export default interface Registry {
+    'SafeTools::SchedulePaymentFormActionCard': typeof SafeToolsSchedulePaymentFormActionCard;
```
Comment (backspace) [inline packages/safe-tools-client/app/components/safe-tools/schedule-payment-form-action-card/index.gts:178]: Why namespace this?

**Review by backspace — APPROVED** [review]

File: packages/boxel/addon/components/boxel/action-chin/usage.gts (hunk 1 above)
Comment (lukemelia) [reply packages/boxel/addon/components/boxel/action-chin/usage.gts:29]: Merge problem. Will remove.

File: packages/boxel/addon/components/boxel/action-container/usage.gts (hunk 2 above)
Comment (lukemelia) [reply packages/boxel/addon/components/boxel/action-container/usage.gts:67]: Merge problem. Will remove.

File: packages/safe-tools-client/app/components/safe-tools/schedule-payment-form-action-card/index.gts (hunk 3 above)
Comment (lukemelia) [reply packages/safe-tools-client/app/components/safe-tools/schedule-payment-form-action-card/index.gts:178]: I'm OK with not scoping.

**Pull request merged**